src/issueHandling/position.ts

```diff
@@ -3,3 +3,11 @@ export default interface Position {
 	fromIndex: number;
 	toIndex: number;
 }
+
+export function setPosition(from: number, to: number) {
+	const position: Position = {
+		fromIndex: from,
+		toIndex: to,
+	};
+	return position;
+}
```

Comment:
I don't really see the added benefit in this function - it's basically the same whether I type `setPosition(5, 8)` or ` `{ fromIndex: 5, toIndex: 8 }`. 
Additionally, the name suggests you're actually changing something by calling the method, like a setter. A better name would be `createPosition` or `newPosition`.

Reply:
This method is still not adding anything anywhere.